website/content/docs/nia/api/tasks.mdx

```diff
@@ -18,15 +18,30 @@ This endpoint allows patch updates to specifically supported fields for existing
 
 #### Request Parameters
 
-* `run` - (string) Values can be only be "inspect" and "now".
-  * "inspect": Does not update the task but returns information on if/how resources would be changed for the proposed task update.
-  * "now": Updates the task accordingly and immediately runs the task, rather than allowing the task to run at the natural daemon cadence
+| Name | Type	 | Required | Description  |
+| -----| ----- | -------- | ------------ |
+|`run` | string | false   | Values can be only be `"inspect"` and `"now"`.<br/><ul><li>`"inspect"`: Does not update the task but returns information on if/how resources would be changed for the proposed task update.</li><li> `"now"`: Updates the task accordingly and immediately runs the task, rather than allowing the task to run at the natural daemon cadence</li></ul>
+
+#### Request Body
+
+| Name | Type	 | Required | Description  |
+| -----| ----- | -------- | ------------ |
+|`enabled` | boolean | true | Whether the task is enabled to run and manage resources.|
+
+#### Response Statuses
+
+| Status | Reason           |
+| ------ | ------------------ |
+| 200  | Successfully updated the task |
+| 404  | Task with the given name not found |
 
 #### Response Fields
 
-* `inspect` - Information on how resources would be changed given a proposed task update, similar to [inspect-mode](/docs/nia/cli/cli-overview#inspect-mode). This is only returned when passed the `run=inspect` request parameter
-  * `changes_present` - (bool) Whether or not changes were detected for running the proposed task update
-  * `plan` - (string) The Terraform plan generated for the proposed task update . Note: a non-empty string does not necessarily indicate changes were detected.
+| Name	| Type	 | Description        |
+| --------- | ------ | ------------------ |
+|`inspect` | object |  Information on how resources would be changed given a proposed task update, similar to [inspect-mode](/docs/nia/cli/cli-overview#inspect-mode). This is only returned when passed the `run=inspect` request parameter.
```

Comment:
```suggestion
|`inspect` | object |  Information on how resources would be changed given a proposed task update, similar to [inspect-mode](/docs/nia/cli/cli-overview#inspect-mode). This is only returned when passed the `run=inspect` query parameter.
```

Reply:
We've been referring to this as request parameter in the rest of the docs, which would be my only hesitation for changing this.

https://www.consul.io/docs/nia/api/tasks#request-parameters

Reply:
Oh okay, no worries then, consistency is better.